TemplateApplication/Onboarding/Consent.swift

```diff
@@ -37,7 +42,13 @@ struct Consent: View {
                 if !FeatureFlags.disableFirebase {
                     onboardingSteps.append(.accountSetup)
                 } else {
-                    onboardingSteps.append(.healthKitPermissions)
+                    if !healthKitDataSource.authorized {
```

Comment:
We should also check `HKHealthStore.isHealthDataAvailable()` here.

Reply:
Good suggestion, we already do that in the `AccountSetup.swift` as well 👍